src/components/settings/SettingsLMSTab/LMSConfigs/SAP/SAPConfigEnablePage.tsx

```diff
@@ -14,19 +14,19 @@ import type {
 
 export const formFieldNames = {
   DISPLAY_NAME: "displayName",
-  SAP_BASE_URL: "sapBaseUrl",
-  SAP_COMPANY_ID: "sapCompanyId",
-  SAP_USER_ID: "sapUserId",
-  OAUTH_CLIENT_ID: "oauthClientId",
-  OAUTH_CLIENT_SECRET: "oauthClientSecret",
-  SAP_USER_TYPE: "sapUserType",
+  SAPF_BASE_URL: "sapsfBaseUrl",
+  SAPF_COMPANY_ID: "sapsfCompanyId",
+  SAPF_USER_ID: "sapsfUserId",
```

Comment:
nit: If we're wanting to be consistent in naming
```suggestion
  SAPSF_BASE_URL: "sapsfBaseUrl",
  SAPSF_COMPANY_ID: "sapsfCompanyId",
  SAPSF_USER_ID: "sapsfUserId",
```